dvc/commands/plots.py

```diff
@@ -75,7 +75,9 @@ def run(self):
                 out if self.args.json else os.path.join(out, "static")
             )
             renderers = match_renderers(
-                plots_data=plots_data, out=renderers_out
+                plots_data=plots_data,
+                out=renderers_out,
+                templates_dir=self.repo.plots.templates_dir,
```

Comment:
Also, it'd be better to push more of this logic inside the API so that you can test them easily.

Reply:
Sorry for my lack of understanding, could you elaborate on what does "the API" refers to?